libcodechecker/analyze.py

```diff
@@ -292,6 +296,11 @@ def main(args):
     with open(metadata_path, 'w') as metafile:
         json.dump(metadata, metafile)
 
+    # WARN: store command will search for this file!!!!
+    compile_cmd_json = os.path.join(args.output_path, 'compile_cmd.json')
+    if not args.logfile[0] == compile_cmd_json:
```

Comment:
- The two paths should be converted to absolute path, because if relative path is given then the two strings may not be equal even for the same paths.
- `a != b` should be used instead of `not a == b`, because this way negation applies to `a`.
- What if the target directory already contains a `compile_cmd.json` file? It won't be overwritten and it can be confusing that the existing JSON file will be used.